### PR TITLE
SENT-33 Initialize platformio project and create esp32 capability tests

### PR DESCRIPTION
**Pull request by Victor-Leung-CA**
Files run correctly and can flash onto my ESP32. PlatformIO project file is configured to work with the esp32 wrover variant and has multiple environments to accommodate flashing different setups.

I have encapsulated some modem functions into a library. (On/off/restart)

Tests that work:
- Basic blink
- LTE connection
- Sending SMS (tinyGSM library does not support receiving SMS, would have to try out other libraries)

### CODE REVIEW

File: embedded/Sentinel Embedded/platformio.ini [hunk 1]
```diff
@@ -0,0 +1,34 @@
+; PlatformIO Project Configuration File
```
Comment (Victor-Leung-CA) [inline embedded/Sentinel Embedded/platformio.ini:1]: Different envs to support flashing different builds.

File: embedded/Sentinel Embedded/platformio.ini [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env]
+platform = espressif32
+board = esp-wrover-kit
+framework = arduino
+monitor_speed = 115200
+upload_port = /dev/cu.wchusbserial54350352381
```
Comment (Victor-Leung-CA) [inline embedded/Sentinel Embedded/platformio.ini:16]: My own upload port, this would be different for others using it.

File: README.md [hunk 3]
```diff
@@ -1 +1 @@
-# sentinel
\ No newline at end of file
+# Sentinel
```
Comment (Victor-Leung-CA) [inline README.md:1]: Maybe we could have a tagline or smth...

File: embedded/Sentinel Embedded/platformio.ini (hunk 2 above)
Comment (KevinetPeng) [reply embedded/Sentinel Embedded/platformio.ini:16]: Might be a dumb question, I'm not super familiar with C++, but would it be possible to set this with an environment variable, or is that too much of a hassle?

File: README.md (hunk 3 above)
Comment (KevinetPeng) [reply README.md:1]: I agree we should come up with something in our next meeting

File: embedded/Sentinel Embedded/platformio.ini (hunk 2 above)
Comment (Victor-Leung-CA) [reply embedded/Sentinel Embedded/platformio.ini:16]: Its a platformIO sepcific environment you can just change in VSCode, takes like 1 second - a button click.

**Review by KevinetPeng — APPROVED** [review]

**Pull request merged**